### PR TITLE
Issue 874 - allow discussion to be closed and/or hidden

### PR DESCRIPTION
**Pull request by sethetter**
* New column on the docs table for `discussion_state`
* Ability in admin panel to set this state
* API endpoint for `PUT /api/docs/:id`, setting stage for further route cleanup in the future
* Indicator on homepage list if discussion state is *not open*.
* Documents with non-open discussion are not listed in "Active Legislation"

Discussion states:
* `closed` - discussion is visible but no new comments / annotations are allowed
* `hidden` - discussion is closed, and current comments / annotations are not viewable
* `open` - all normal, discussion viewable, new discussion allowed

@krues8dr

### CODE REVIEW

**Comment by krues8dr** [comment]
Nice! :+1:

**Comment by emanuelfeld** [comment]
@sethetter there appears to be an issue in the discussion state menu (drop down menu also has no options)

image

cc @mattbailey0

**Comment by sethetter** [comment]
@emanuelfeld try clearing cache. I know it's less than ideal but we haven't yet addressed the cache busting of the locales file.

Give that a shot and let me know if the issue persists. It looks good on my end.

**Comment by emanuelfeld** [comment]
Works, thanks @sethetter

**Comment by sethetter** [comment]
@emanuelfeld :metal:

File: app/Http/Controllers/DocumentController.php [hunk 1]
```diff
@@ -153,6 +154,15 @@ public function postDocs()
         }
     }
 
+    public function update($id, UpdateDocumentRequest $request)
+    {
+        file_put_contents('/var/log/madison_debug', '?'."\n\n", FILE_APPEND);
```
Comment (krues8dr) [inline app/Http/Controllers/DocumentController.php]: @sethetter should this still be in here?

Comment (sethetter) [inline app/Http/Controllers/DocumentController.php]: Left it in by mistake but it's since been removed.

**Pull request merged**